test/features/edgeconnect/edgeconnect.go

```diff
@@ -49,13 +49,17 @@ func NormalModeFeature(t *testing.T) features.Feature {
 
 	edgeConnectTenantConfig := &tenantConfig{}
 
-	builder.Assess("create EC configuration on the tenant", createTenantConfig(secretConfig, edgeConnectTenantConfig))
+	randSuffix := uuid.NewString()[:defaultRandomLength]
+	testECname := "test-edgeconnect-normal-" + randSuffix
+	testHostPattern := fmt.Sprintf("e2eTestHostPattern%s.internal.org", randSuffix)
+
```

Comment:
why overcomplicate it with only using part of the uuid? also kinda defeats the purpose of the uuid 😅 

```
	testECname := uuid.NewString()
	testHostPattern := fmt.Sprintf("e2eTestHostPattern.%s.internal.org", testECname)
```

the name `test-edgeconnect-normal-` serves no real purpose, and you can perfectly see what was created by the e2e tests from the logs: (also the ec tests are so fast, that you will not see the deployment if you are not directly looking at it 😅 )
```
--- PASS: TestStandard (46.06s)
    --- PASS: TestStandard/edgeconnect-install (13.18s)
        --- PASS: TestStandard/edgeconnect-install/create_EC_configuration_on_the_tenant (1.07s)
        --- PASS: TestStandard/edgeconnect-install/create_client_secret (0.28s)
        --- PASS: TestStandard/edgeconnect-install/'0b038866-29ce-4c9e-b8eb-dc77791c6d18'_edgeconnect_created (0.30s)
        --- PASS: TestStandard/edgeconnect-install/'0b038866-29ce-4c9e-b8eb-dc77791c6d18'_edgeconnect_phase_changes_to_'Running' (5.15s)
        --- PASS: TestStandard/edgeconnect-install/check_EC_configuration_on_the_tenant (0.39s)
        --- PASS: TestStandard/edgeconnect-install/delete_EdgeConnect_CR (5.28s)
        --- PASS: TestStandard/edgeconnect-install/check_if_EC_configuration_is_deleted_on_the_tenant (0.28s)
    --- PASS: TestStandard/edgeconnect-install-provisioner (17.65s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/create_edgeconnect_client_secret (0.13s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/'8b312735-b051-42ff-98f4-18a693865d43'_edgeconnect_created (0.13s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/'8b312735-b051-42ff-98f4-18a693865d43'_edgeconnect_phase_changes_to_'Running' (5.16s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/get_tenant_config (0.25s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/get_EC_status (0.13s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/check_if_EC_configuration_exists_on_the_tenant (0.27s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/check_hostPatterns_on_the_tenant_-_testHostPattern (0.35s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/update_hostPatterns (5.28s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/check_hostPatterns_on_the_tenant_-_testHostPattern2 (0.28s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/delete_EC_custom_resource (5.29s)
        --- PASS: TestStandard/edgeconnect-install-provisioner/check_if_EC_configuration_is_deleted_on_the_tenant (0.25s)
    --- PASS: TestStandard/edgeconnect-install-k8s-automation (13.04s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/create_ServiceAccount (0.14s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/create_edgeconnect_client_secret (0.13s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/'7ba14241-09b0-4d07-a8b7-cab65118d65c'_edgeconnect_created (0.13s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/'7ba14241-09b0-4d07-a8b7-cab65118d65c'_edgeconnect_phase_changes_to_'Running' (5.14s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/get_tenant_config (0.25s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/get_EC_status (0.13s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/check_if_EC_configuration_exists_on_the_tenant (0.27s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/check_hostPatterns_-_k8s_automation (0.34s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/check_if_settings_object_exists_on_the_tenant (0.39s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/delete_EC_custom_resource (5.28s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/check_if_EC_configuration_is_deleted_on_the_tenant (0.25s)
        --- PASS: TestStandard/edgeconnect-install-k8s-automation/check_if_settings_object_is_deleted_on_the_tenant (0.30s)
PASS
out: helm uninstall dynatrace-operator \
			--namespace dynatrace
release "dynatrace-operator" uninstalled

ok  	github.com/Dynatrace/dynatrace-operator/test/scenarios/standard	129.859s
```

Reply:
yeah it was my idea too, let's go with full uuid names. :)

Reply:
fixed in 3af244ec